test/Microsoft.Extensions.Logging.Testing.Tests/project.json

```diff
@@ -14,12 +16,16 @@
     "dnx451": {
```

Comment:
We can probably start changing this to net451 since we're using the console runner